<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdventureCardEdit</class>
 <widget class="QWidget" name="AdventureCardEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AdventureCardEdit</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblCard">
        <property name="text">
         <string>Card</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblTitle">
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblType">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblText">
        <property name="text">
         <string>Text</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblEncounterNumber">
        <property name="text">
         <string>Encounter Number</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblSubType">
        <property name="text">
         <string>Sub Type</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lblAttackPoints">
        <property name="text">
         <string>Attack Points</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cboCard"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtTitle"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtType"/>
      </item>
      <item row="3" column="1">
       <widget class="QTextEdit" name="txtText"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txtEncounterNumber"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="txtSubType"/>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="txtAttackPoints"/>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="btnReload">
        <property name="text">
         <string>Reload</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
